.changeset/curly-zoos-exist.md

```
@@ -0,0 +1,5 @@
+---
+"template-files": minor
+---
+
+Add new special file treatment: delete (path still neede nonetheless, altough useless)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)